source/_integrations/default_config.markdown

```diff
@@ -37,7 +37,7 @@ This integration is a meta-component and configures a default set of integration
 - [Logbook](/integrations/logbook/) (`logbook`)
 - [Logger](/integrations/logger/) (`logger`)
 - [Map](/integrations/map/) (`map`)
-- [Media Source](/integrations/media_source/) (`media_source`)
+- [Media source](/integrations/media_source/) (`media_source`)
 - [Mobile App Support](/integrations/mobile_app/) (`mobile_app`)
```

Comment:
```suggestion
- [Mobile app support](/integrations/mobile_app/) (`mobile_app`)
```

Reply:
`[System Health](/integrations/system_health/) (`system_health`)` as well, but its too far outside of touched files, so I cant suggest it.

Reply:
Oh good one